src/components/common/RouterLinkNetwork.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <router-link
+    :class="routerClass"
+    :to="url"
+    :exact="exact"
+    :title="title"
+    @click.native="handleClick()"
+  >
+    <slot></slot>
+  </router-link>
+</template>
+
+<script>
+import { mapGetters } from "vuex"
+export default {
+  name: `routerLinkNetwork`,
+  props: {
+    routerClass: {
+      type: String,
+      default: null
+    },
+    to: {
+      type: String,
+      default: null
+    },
+    exact: {
+      type: String,
+      default: null
+    },
+    title: {
+      type: String,
+      default: null
+    }
+  },
+  computed: {
+    ...mapGetters([`network`]),
+    url() {
+      if (this.network.length) {
```

Comment:
is `network` an array?

Reply:
Nope, just checking if it's not empty

Reply:
well then not use `length` but check if it is empty. this code suggests it is an array...